prow/overlays/smaug/config.yaml

```diff
@@ -372,7 +372,71 @@ tide:
           probot-template:
             skip-unknown-contexts: false
 
+presets:
+- labels:
+  preset-xdg-cache: "true"
+  env:
+    - name: XDG_CACHE_HOME
+      value: /var/cache/xdg
+  volumesMount:
+    - name: cache
+      mountPath: /var/cache/xdg
+  volumes:
+    - name: cache
+      persistentVolumeClaim:
+        claimName: ci-xdg-cache
+
+presubmits:
+  thoth-station:
```

Comment:
we dont need this function here,
we tend to use the in_repo config with .prow.yaml for this.